dll/win32/windowscodecs/main.c

```diff
@@ -228,6 +228,28 @@ void reverse_bgr8(UINT bytesperpixel, LPBYTE bits, UINT width, UINT height, INT
     }
 }
 
+void convert_rgba_to_bgra(UINT bytesperpixel, LPBYTE bits, UINT width, UINT height, INT stride)
+{
+    UINT x, y;
+    BYTE *pixel, temp;
+
+    for (y=0; y<height; y++)
+    {
+        pixel = bits + stride * y;
+
+        for (x=0; x<width; x++)
+        {
+            temp = pixel[3];
+            pixel[3] = pixel[0];
+            pixel[0] = pixel[1];
+            pixel[1] = pixel[2];
+//            pixel[2] = pixel[3];
```

Comment:
Remove this comment line.

Reply:
I can if need, no big deal, but it is a better reminder then writing a ton of words to see what was done and why temp had to be used. I screwed up the conversion, because at first attempt i did not take into account that i need temp and order is also important. But if others recommend it too, i will do a new commit.

Reply:
The code is fairly obvious in this regard, the commented line seems more like a testing leftover now.

Reply:
Done